src/posts/xarray-kvikio/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: 'Enabling GPU-native analytics with Xarray and kvikIO'
+date: '2022-08-25'
```

Comment:
Need to update this date before publishing.

Do we want to publish the blog post soon-ish (e.g. by end of August) or wait for a bit longer (e.g. until some of those WIP PRs get a bit more stable)? Btw, I don't have merge permissions on this repo...

Reply:
@weiji14, i updated the date. Regarding whether we should wait or ship this as is, i'll defer to you :)

Reply:
Let's ship it :rocket: